fetch_links.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Apr  5 23:47:27 2020
+
+@author: agato
+"""
+
+import urllib.request
+from bs4 import BeautifulSoup
+
+url = input("Input your url: ")
+#url = 'file:///C:/Users/agato/Desktop/example3.html'
+response = urllib.request.urlopen(url)
+data = response.read()
+doc = BeautifulSoup(data, 'html.parser')
+
+li = doc.select('ol > li')
```

Comment:
Because you're getting all `<li>` elements in one go, if you have more than one list on the page, the numbering will continue. So if you have lists like so:
```
1. one
2. two
3. three

1. one
2. two
3. three
```

your script will output:
```
1. one
2. two
3. three
4. one
5. two
6. three
```
Instead, you need to get all the `<ol>` nodes first and then **for each** such node print out the contents of the `<li>` nodes with proper numbering.

Consider using the following:
```
lists = doc.find_all('ol')
for list in lists:
   list_items = list.find_all('li')
```